lib/vmdb/global_methods.rb

```diff
@@ -67,6 +67,23 @@ def format_timezone(time, timezone = Time.zone.name, ftype = "view")
       new_time
     end
 
+    # Create a time in a timezone, return in UTC
+    def create_time_in_utc(datetime, tz = nil)                # tz = nil means use user's session timzone
```

Comment:
I would prefer if we didn't have to add more global methods...I wonder if there's a better place we can drop this so it's available to both controllers and the Report::Timer class.  Perhaps a mixin that is included in both locations?  Not sure where would be best for that mixin to live...maybe helpers/mixins or controllers/mixins?